src/sqlalchemy_cratedb/compiler.py

```diff
@@ -25,6 +25,7 @@
 
 import sqlalchemy as sa
 from sqlalchemy.dialects.postgresql.base import PGCompiler
+from sqlalchemy.dialects.postgresql.base import RESERVED_WORDS as POSTGRESQL_RESERVED_WORDS
```

Comment:
I think we have some deviations from the postgres reserved keywords, I think it's better if we used the python sqlparser to get the list for CrateDB programmatically.

Reply:
We can do it on a later iteration if we agree. I am not sure if it would be too heavy, see https://github.com/crate/sqlalchemy-cratedb/pull/21#discussion_r1628501620.

Reply:
yep, we can do in next step.

Reply:
Thanks! Maybe you want to submit a corresponding patch, @surister?